app/components/dashboard/dashboard.component.js

```diff
@@ -58,12 +59,16 @@ class Dashboard extends React.Component {
       !error && (
         <div className="dashboard">
           <Segment loading={isLoading} className="secondary">
-            <div className="counter-uk">
-              <div className="wrapper">
-                <span className="text">Overal Distance: </span>
-                <span className="total">{total}km</span>
+            <div className="heading">
+              <div className="counter-uk">
+                <div className="wrapper">
+                  <span className="text">Overal Distance: </span>
+                  <span className="total">{total}km</span>
```

Comment:
For some reason my total value is to many decimal places unlike the photos you shared on the team channel. Not sure why, added screenshot
![Screenshot 2021-05-26 at 16 31 03](https://user-images.githubusercontent.com/80352253/119688550-fbb39700-be3f-11eb-9fe7-d2e561af7e47.png)